README.md

```diff
@@ -17,69 +15,90 @@ Could be run on earlier versions of Django, but expect some missing imports warn
 pip install django-stubs
 ```
 
-
-## Mypy compatibility
-
-| django-stubs | mypy version | django version | python version
-| ------------ | ---- | ---- | ---- |
-| 1.3.0 | 0.750 | 2.2.x | ^3.6
-| 1.2.0 | 0.730 | 2.2.x | ^3.6
-| 1.1.0 | 0.720 | 2.2.x | ^3.6
-| 0.12.x | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
+See [Configutation](#configuration) section to get started.
 
 
 ## Configuration
 
-To make mypy aware of the plugin, you need to add
+To make `mypy` happy, you will need to add:
 
 ```ini
 [mypy]
 plugins =
     mypy_django_plugin.main
+    
+[mypy.plugins.django-stubs]
+django_settings_module = "myproject.settings"
 ```
 
 in your `mypy.ini` or `setup.cfg` [file](https://mypy.readthedocs.io/en/latest/config_file.html).
 
-Plugin also requires Django settings module (what you put into `DJANGO_SETTINGS_MODULE` variable) to be specified.
+Two things happeining here:
 
-```ini
-[mypy]
-strict_optional = True
+1. We need to explicitly list our plugin to be loaded by `mypy`
+2. Our plugin also requires `django` settings module (what you put into `DJANGO_SETTINGS_MODULE` variable) to be specified
+
+This fully working [typed boilerplate](https://github.com/wemake-services/wemake-django-template) can serve you as an example.
 
-# This one is new:
-[mypy.plugins.django-stubs]
-django_settings_module = mysettings
-```
 
-Where `mysettings` is a value of `DJANGO_SETTINGS_MODULE` (with or without quotes)
+## Version compatibility
+
+We rely on different `django` and `mypy` versions:
+
+| django-stubs | mypy version | django version | python version
+| ------------ | ---- | ---- | ---- |
+| 1.3.0 | 0.750 | 2.2.x | ^3.6
+| 1.2.0 | 0.730 | 2.2.x | ^3.6
+| 1.1.0 | 0.720 | 2.2.x | ^3.6
+| 0.12.x | old semantic analyzer (<0.711), dmypy support | 2.1.x | ^3.6
+
+
+## FAQ
```

Comment:
Today @mkurnikov provided a good explanation on how to add types to a custom field https://gitter.im/mypy-django/Lobby?at=5e38870140da694c5e06a6ef, it looks like a good candidate for FAQ, another one would be "how to make mypy use custom User model in request.user" the answer is to override original request type with:

```
class MyRequest(Request):
    user: MyUser
```
and use it throughout codebase.

Reply:
Great feedback! Thanks!

Reply:
Done!